packages/react-vapor/src/components/table-hoc/TableHOC.tsx

```diff
@@ -35,7 +35,7 @@ export class TableHOC extends React.PureComponent<ITableHOCProps & React.HTMLAtt
                 })}
             >
                 {this.renderActions()}
-                <table className={classNames(this.props.className)}>
+                <table className={classNames(this.props.className)} id={this.props.id}>

```

Comment:
Since `ActionBarConnected` down here also has the same `this.props.id`, there is a slight possibility that the page would contain the same `id` element twice, for instance, if `ActionBar` is updated to actually output this ID.

I think either or both of those IDs should be prefixed, just to be sure :)

Reply:
Yes, there is indeed two id="complex-example" (in the local demo); one for the table and for the actionBar, but i don't understand what you mean by prefixing the IDs :S

Reply:
`prefixing` => I mean adding a prefix, like `table-` :)

`id={`table-${this.props.id}`}`